src/openforms/formio/tests/test_validators.py

```diff
@@ -164,6 +164,16 @@ def test_heic_brand_heif_files_are_recognized_as_heic(self):
         except ValidationError as e:
             self.fail(f"Valid file failed validation: {e}")
 
+    def test_heic_brand_heif_files_are_not_recognized_as_png(self):
+        # lib magic has a hard time recognizing the HEVC is used and a heif container actutally is heic
+        validator = validators.MimeTypeValidator({"image/png"})  # accept any
+        sample_2 = SimpleUploadedFile(
+            "sample2.heic", b"\x00\x00\x00\x18ftypheic", content_type="image/heif"
+        )
+
+        with self.assertRaises(ValidationError):
+            validator(sample_2)
+
```

Comment:
I expected this to return a 400 on TemporaryFileUpload. But it doesn't.
So right now you can upload a heic in a "image/png" field and on going to the next step it will mark the field as invalid. :shrug: Works. But I had the wrong expectation myself.